app/views/handlers/individual_response.py

```diff
@@ -0,0 +1,307 @@
+                "text": lazy_gettext(
+                    "Do you want to send an individual access code to <em>{person_name}</em> by post?"
+                ),
+                "placeholders": IndividualResponseHandler._person_name_placeholder,
+            },
+            "description": lazy_gettext(
+                "<p>A letter with an indivdual access code will be sent to your registered household address</p>"
+            ),
+            "guidance": {
+                "contents": [
+                    {
+                        "description": lazy_gettext(
+                            "The letter will be addressed to <strong>Individual Resident</strong> instead of the name provided"
+                        )
+                    }
+                ]
+            },
+            "answers": [
+                {
+                    "type": "Radio",
+                    "id": "individual-response-post-confirm-answer",
+                    "mandatory": True,
+                    "options": [
+                        {
+                            "label": lazy_gettext("Yes, send the access by post"),
+                            "value": "Yes, send the access by post",
+                        },
+                        {
+                            "label": lazy_gettext("No, send it another way"),
+                            "value": "No, send it another way",
+                        },
+                    ],
+                }
+            ],
+        },
+    }
+
+    def __init__(self, schema, questionnaire_store, language, form_data, list_item_id):
+        super().__init__(
+            self.block_definition,
+            schema,
+            questionnaire_store,
+            language,
+            form_data,
+            list_item_id,
+        )
+
+    @cached_property
+    def answer_id(self):
+        return self.rendered_block["question"]["answers"][0]["id"]
+
+    @cached_property
+    def confirm_option(self):
+        return self.rendered_block["question"]["answers"][0]["options"][0]["value"]
+
+    @cached_property
+    def selected_option(self):
+        return self.form.get_data(self.answer_id)
+
+    def handle_get(self):
+        previous_location_url = url_for(
+            "individual_response.get_individual_response_how",
+            list_item_id=self._list_item_id,
+        )
+
+        return render_template(
+            "individual_response/how",
+            language=self._language,
+            content=self.get_context(),
+            previous_location_url=previous_location_url,
+        )
+
+    def handle_post(self):
+        if self.selected_option == self.confirm_option:
+            self._questionnaire_store.progress_store.update_section_status(
+                CompletionStatus.INDIVIDUAL_RESPONSE_REQUESTED,
+                self._schema.json["individual_response"]["individual_section_id"],
+                self._list_item_id,
+            )
+            self._questionnaire_store.save()
```

Comment:
The progress store `is_dirty` flag should be checked here to see if saving is necessary.